src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -158,7 +174,7 @@ function prepareContribQuestions() {
         var qnNum = contribQuestionNums[i];
 
         // Get number of options for the specified question number of contribution question type
-        var optionNums = $('[name^="responsetext-' + qnNum + '-"]').length;
+        var optionNums = $('[name^="responsetext-' + qnNum + '"]').length;
```

Comment:
@tania, I think you accidentally reverted some changes in an earlier pull request here?